docs/source/conf.py

```diff
@@ -13,7 +13,7 @@
 # import os
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
-import sphinx_rtd_theme
+# import sphinx_rtd_theme
```

Comment:
Commented on this line due to the failure of flake8 on this file:
![image](https://user-images.githubusercontent.com/52296418/95766405-d6b74d00-0cd0-11eb-907c-c5bb1359fb30.png)

Reply:
hey, hi again. You did right to check flask before submitting the PR. However, in this case, this import is required to create the documentation properly. Travis build would still work with that because the `docs` folder is ignored in `.dockerignore`. To avoid to get this error again, I suggest you to modify `.flake8.yml` and add `docs` in the exclusion pattern because we do not want to lint those files too. Then, you can add again that import. Thank you!